lib/htmlGen.js

```diff
@@ -30,6 +30,11 @@ var htmlGen = {
     listItem.innerHTML = noteSummary;
     document.getElementById("ls").appendChild(listItem);
   },
+  createDiv: function(args = {id: "notes"}){
+    var div = document.createElement("div");
+    this._addAttributes(div, args);
+    document.body.appendChild(div);
```

Comment:
Can we change "document.body" here to "target" instead, similar to the other functions in HTMLgen? Makes it more re-usable if needed.